doc/TBG_macros.cfg

```diff
@@ -188,6 +188,19 @@ TBG_restart="--restart"
 # Note: does currently not work with `Radiation` plugin
 TBG_softRestarts="--softRestarts 5000"
 
+# ISAAC server and rendering interval settings
+TBG_isaac="--isaac.period 1 --isaac.name !TBG_jobName --isaac.url <server_url>"
+
+# ISAAC JPEG quality (1..100). Smaller values are faster sent, but of lower quality
+TBG_isaac_quality="--isaac.quality 80"
```

Comment:
I am sorry, I couldn't read your text because I am currently at a Deutsche Bahn train trying to reach a... .a ... zzz... .. ... but soon vectoring will ... .also ... 640K ... is all the memory anybody ...